modules/persistent-storage-csi-google-cloud-file-delete-instances.adoc

```diff
@@ -3,22 +3,24 @@
 // * storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="persistent-storage-csi-gcp-cloud-file-delete-instances_{context}"]
+[id="persistent-storage-csi-google-cloud-file-delete-instances_{context}"]
 = Destroying clusters and GCP Filestore
 
-Typically, if you destroy a cluster, the {product-title} installer deletes all of the cloud resources that belong to that cluster. However, when a cluster is destroyed, Google Compute Platform (GCP) Filestore instances are not automatically deleted, so you must manually delete all persistent volume claims (PVCs) that use the Filestore storage class before destroying the cluster.
+Typically, if you destroy a cluster, the {product-title} installer deletes all of the cloud resources that belong to that cluster. However, due to the special nature of the Google Compute Platform (GCP) Filestore resources, the automated cleanup process might not remove all of them in some cases. 
+
+Therefore, Red Hat recommends that you verify that all cluster-owned Filestore resources are deleted by the uninstall process.
 
 .Procedure
-To delete all GCP Filestore PVCs:
+To ensure that all GCP Filestore PVCs have been deleted:
```

Comment:
Stupid question, how can you use the oc command after the cluster has been destroyed?
cc @tsmetana

Reply:
Good question!

Reply:
You can't. You have to use gcloud (either CLI or the web GUI), I'm afraid.

Reply:
@tsmetana Ugh. I don't have way to emulate that to get the procedure. Who can help me out?

Reply:
@tsmetana I think we're good with saying use google CLI/GUI but is there any label the customers can rely on to filter the filestore resources that were created by the cluster but not removed?

Reply:
Yes, there's the `kubernetes-io-cluster-${CLUSTER_ID}=owned` label -- the cluster ID is unique to the cluster, so there should not exist any filestores owned byt the deleted cluster after the uninstallation.

Reply:
> @tsmetana Ugh. I don't have way to emulate that to get the procedure. Who can help me out?

I agree with @gcharot, just mentioning to use the GCloud tools to verify the resources with the deleted cluster ID label are gone should be enough. I think in real-world scenarios the failure to wipe the filestores should be very rare.

Reply:
Done.